x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_management/classic.tsx

```diff
@@ -36,22 +38,42 @@ export function ClassicStreamDetailManagement({
 
   if (!definition.data_stream_exists) {
     return (
-      <EuiFlexGroup direction="column">
-        <EuiCallOut
-          title={i18n.translate('xpack.streams.unmanagedStreamOverview.missingDatastream.title', {
-            defaultMessage: 'Data stream missing',
-          })}
-          color="danger"
-          iconType="error"
-        >
-          <p>
-            {i18n.translate('xpack.streams.unmanagedStreamOverview.missingDatastream.description', {
-              defaultMessage:
-                'The underlying Elasticsearch data stream for this classic stream is missing. Recreate the data stream to restore the stream by sending data before using the management features.',
+      <>
+        <StreamsAppPageTemplate.Header
+          bottomBorder="extended"
+          pageTitle={
+            <EuiFlexGroup gutterSize="s" alignItems="center">
+              {i18n.translate('xpack.streams.entityDetailViewWithoutParams.manageStreamTitle', {
+                defaultMessage: 'Manage stream {streamId}',
+                values: { streamId: key },
+              })}
+              <EuiBadgeGroup gutterSize="s">
+                {isUnwiredStreamDefinition(definition.stream) && <ClassicStreamBadge />}
+                <LifecycleBadge lifecycle={definition.effective_lifecycle} />
+              </EuiBadgeGroup>
+            </EuiFlexGroup>
+          }
+        />
+        <StreamsAppPageTemplate.Body>
+          <EuiCallOut
+            title={i18n.translate('xpack.streams.unmanagedStreamOverview.missingDatastream.title', {
+              defaultMessage: 'Data stream missing',
```

Comment:
I was thinking a prompt instead of a callout but both could work.

<img width="1253" alt="image" src="https://github.com/user-attachments/assets/6edb0816-d0c9-41f2-88eb-826948edc968" />


<img width="862" alt="image" src="https://github.com/user-attachments/assets/48d35d0d-f3b6-47b9-957f-44fd28bb0033" />